src/accidental.ts

```diff
@@ -17,38 +17,75 @@ import { Modifier } from './modifier';
 import { Glyph } from './glyph';
 import { GraceNoteGroup } from './gracenotegroup';
 import { GraceNote } from './gracenote';
+import { ModifierContextState } from './modifiercontext';
+import { Voice } from './voice';
+import { Note } from './note';
+import { StaveNote } from './stavenote';
+
+export interface Line {
+  column: number;
+  line: number;
+  flatLine: boolean;
+  dblSharpLine: boolean;
+  numAcc: number;
+  width: number;
+}
+
+export interface AccidentalListItem {
```

Comment:
This can just be an internal type alias. No need for interface.

Reply:
done